.github/workflows/deploy-enterprise.yml

```diff
@@ -60,8 +60,10 @@ jobs:
             default_credits_per_user: 0
             default_application_status: APPROVED
             cost_profile: DEFAULT
-            job_files: job_spec/INSAR_ISCE.yml
-            instance_types: c6id.xlarge,c6id.2xlarge,c6id.4xlarge,c6id.8xlarge
+            job_files: >-
+              job_spec/ARIA_RAIDER.yml
+              job_spec/INSAR_ISCE_TEST.yml
```

Comment:
Note: INSAR_ISCE and INSAR_ISCE_TEST only differ by the `++omp-num-threads` argument right now, and test doesn't override the image tag

Reply:
Shouldn't we modify that so the correct number of jobs fit?
https://github.com/ASFHyP3/hyp3/blob/develop/job_spec/INSAR_ISCE_TEST.yml#L100

Looks like we might put too many jobs on an instance without modifying this too...

Reply:
@cmarshak this is how `hyp3-a19-jpl-test` is configured, see:
https://github.com/ASFHyP3/hyp3/blob/develop/.github/workflows/deploy-enterprise-test.yml#L41-L58

Reply:
I understand now - thanks!